src/cms/templates/pois/poi_list.html

```diff
@@ -38,9 +38,9 @@ <h1 class="heading">{% trans 'Locations on map' %}</h1>
     <table data-enable-table-sort class="sort w-full mt-4 rounded border border-gray-200 shadow bg-white table-fixed">
         <thead>
             <tr class="border-b border-gray-200">
-                <th class="text-sm text-left uppercase py-3 pl-4 pr-2 min">{% trans 'ID' %}</th>
-                <th class="text-sm text-left uppercase py-3 pr-2 min">{% trans 'Version' %}</th>
-                <th class="text-sm text-left uppercase py-3 pr-2">{% trans 'Status' %}</th>
+                <!--<th class="text-sm text-left uppercase py-3 pl-4 pr-2 min">{% trans 'ID' %}</th>
+                <th class="text-sm text-left uppercase py-3 pr-2 min">{% trans 'Version' %}</th>-->
```

Comment:
```suggestion
```
Please do not comment out code, but just delete it. The whole code history is stored in git, so nothing can be lost forever if the lines are deleted.